### ISSUE TITLE
`/me` or `/user` endpoint

### ISSUE THREAD

**Issue by jessfraz** [oxidecomputer/omicron#596]
something super basic that tells us info about the current authed user

you can pick your poison on the name, github uses `/user` but I've seen `/me` as well

I lean towards `/user`

**Comment by david-crespo** [comment]
Exists already, though we don't have a `User` model so it basically only returns the current user's UUID.

https://github.com/oxidecomputer/omicron/blob/72b8c3ec24a82e2778099e818025b91f335186c8/nexus/src/external_api/console_api.rs#L203-L222

**Comment by jessfraz** [comment]
oh perfecrt somehow i missed that in the spec would be nice to have more info even if it is slightly stale from the idp

basically my use case is when the user wants to get their auth status from the cli I want to return this info like name, email, basic stuff so they know who they are logged in as

**Comment by davepacheco** [comment]
Yeah, we'll definitely put more information there once we implement IdP integration.

**Comment by jessfraz** [comment]
also adding a note here just so we don't forget will also be nice to know the org the user is in, such that then for the cli I will set it as their default org, and check time to time to make sure it is still their org
